src/components/Accordion/Accordion.stories.tsx

```diff
@@ -57,23 +61,48 @@ const Template: ComponentStory<typeof Accordion> = (args) => {
   const AccordionExampleContent =
     'Accordion-innhold uten css for å tilrettelegge for selvvalgt styling';
 
-  const ActionButton = <Button>Separat funksjonsknapp</Button>;
+  const [{ useActions }] = useArgs();
+  const [{ subtitle }] = useArgs();
+
+  const ActionDiv = useActions ? (
+    <div>
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          marginRight: '1rem',
+        }}
```

Comment:
The storybook examples should contain as little css as possible, so that they reflect how the component looks "out of the box". 

```suggestion
```

This style (`align-items:center`) should probably be set directly on the `accordion-header__actions` instead, so that the alignment is correctly set "out of the box"?

Reply:
Agreed 😊